apps/smartling/lambda/package.json

```diff
@@ -35,9 +35,12 @@
     "supertest": "^4.0.2",
     "ts-jest": "^27.1.5",
     "ts-node": "^10.9.1",
-    "tslint": "^5.20.1",
+    "tslint": "^6.1.3",
     "tslint-config-prettier": "^1.18.0",
     "tslint-config-standard": "^9.0.0",
     "tslint-no-unused-expression-chai": "^0.1.4"
+  },
+  "overrides": {
+    "@serverless/dashboard-plugin": "6.1.6"
```

Comment:
as discussed, can we make sure we have a TIMEBOMB backlog ticket to upgrade these lambdas to serverless >= 3 and get rid of this [risky] force override?

Reply:
Yes, just added it to the backlog!